functions/log/package.json

```diff
@@ -9,10 +9,10 @@
     "test": "mocha -R spec -t 120000 --require intelli-espower-loader ../../test/_setup.js test/*.test.js"
   },
   "dependencies": {
-    "gcloud": "^0.37.0",
-    "googleapis": "^12.0.0"
+    "@google-cloud/logging": "^0.1.1",
+    "googleapis": "^12.2.0"
   },
   "devDependencies": {
-    "mocha": "^2.5.3"
+    "mocha": "^3.0.2"
```

Comment:
Can/should this be a higher-level dependency? (so it doesn't have to be in every subfolder's `package.json`)